home/screens/RedesignedProjectScreen/WarningBox.tsx

```diff
@@ -0,0 +1,64 @@
+import { spacing } from '@expo/styleguide-native';
+import { PressableOpacity } from 'components/PressableOpacity';
+import { useExpoTheme, Text, Spacer, Row, View } from 'expo-dev-client-components';
+import * as React from 'react';
+import { Platform } from 'react-native';
+
+import { Ionicons } from '../../components/Icons';
+
+export function WarningBox({
+  title,
+  message,
+  showLearnMore,
+  onLearnMorePress,
+}: {
+  title: string;
+  message: string;
+  showLearnMore?: boolean;
+  onLearnMorePress?: () => void;
+}) {
+  const theme = useExpoTheme();
+
+  const learnMoreButton = showLearnMore ? (
+    <>
+      <Spacer.Vertical size="small" />
+      <PressableOpacity
+        onPress={onLearnMorePress}
+        containerProps={{
+          style: {
+            padding: spacing[2],
+            alignSelf: 'flex-start',
+            backgroundColor: theme.button.tertiary.background,
+          },
+          rounded: 'small',
+        }}>
+        <Text type="InterSemiBold" style={{ color: theme.button.tertiary.foreground }} size="small">
```

Comment:
I think you can also do `<Button.Text color="tertiary" />` if you wanted

Reply:
I think I was having problems with Button.text which is why I did this

Reply:
It was showing the wrong color in dark mode